src/presentation/value-objects/rundown-event.ts

```diff
@@ -4,6 +4,7 @@ import { PartDto } from '../dtos/part-dto'
 import { PieceDto } from '../dtos/piece-dto'
 import { SegmentDto } from '../dtos/segment-dto'
 import { BasicRundownDto } from '../dtos/basic-rundown-dto'
+import {RundownDto} from '../dtos/rundown-dto'
```

Comment:
Spaces around the imported interface would be appreciated. Really should be something that lint handles for us...

Reply:
I try to keep up with it, but until the lint takes care of it we'll see these issues.